Sources/PostgresNIO/PostgresDatabase+Query.swift

```diff
@@ -76,12 +76,13 @@ public struct PostgresQueryMetadata {
             self.rows = Int(parts[2])
         case "DELETE", "UPDATE", "SELECT", "MOVE", "FETCH", "COPY":
             // <cmd> rows
-            guard parts.count == 2 else {
+            // Note, Redshift does not return the actual count as per the spec
+            guard parts.count == 1 || parts.count == 2 else {
                 return nil
             }
             self.command = .init(parts[0])
             self.oid = nil
-            self.rows = Int(parts[1])
+            self.rows = parts.count == 2 ? Int(parts[1]) : 0
```

Comment:
Would you mind refactoring out, the `“SELECT”` into its own case? Right now we would also support `UPDATE` and friends without row number as well.
```swift
case “SELECT” where parts.count == 1:
    // AWS Redshift does not return the actual row count as defined in the postgres wire spec:
    // https://www.postgresql.org/docs/13/protocol-message-formats.html in section `CommandComplete`
    self.command = .init(“SELECT”)
    self.oid = nil
    self.rows = nil
case “DELETE”, “UPDATE”, “SELECT”, “MOVE”, “FETCH”, “COPY”:
    // all the previous code
```

Reply:
Yes, makes sense! Done

Reply:
I did do it different from your comment since I figured if the row count is present, we should parse it. But if that's not important (I really have no idea), I'm happy to just set rows to nil. Please let me know.